util/processinfo_test.go

```diff
@@ -13,52 +13,3 @@
 // limitations under the License.
 
 package util_test
-
-import (
-	"testing"
-
-	"github.com/pingcap/tidb/config"
-	"github.com/pingcap/tidb/util"
-	"github.com/stretchr/testify/assert"
-)
-
-func TestGlobalConnID(t *testing.T) {
```

Comment:
Why remove all tests?

Reply:
Moved to `global_conn_id_test.go`. Because I moved `GlobalConnID` related codes from `processinfo.go` to `global_conn_id.go`.

Reply:
Thanks for your reply. Then you can remove the whole test file as it is blank?

Reply:
OK~

Reply:
> Thanks for your reply. Then you can remove the whole test file as it is blank?

Done. PTAL~